Update SECURITY.md (#9355)
Exiv2/exiv2: SECURITY.md

The draft security advisory is private until we publish it, so it is a good place to discuss the details of the vulnerability privately.

> To qualify as a security issue, the bug **must** be reproducible on an official release of Exiv2, via a realistic attack vector. That means it should be reproducible with a simple command like: `exiv2 poc-file`. And it needs to cause something genuinely bad to happen, like an out-of-bounds memory write. We also always treat OSS-Fuzz crashes (see [our OSS-Fuzz configuration](https://github.com/google/oss-fuzz/tree/master/projects/exiv2) for build instructions) as security issues.

Author: ```suggestion
To qualify as a security issue, the bug **must** be reproducible on an official release of Exiv2, via a realistic attack vector. That means it should be reproducible with a simple command like: `exiv2 poc-file`. And the reproduction steps need to cause something genuinely bad to happen, like an out-of-bounds memory write. We also always treat OSS-Fuzz crashes (see [our OSS-Fuzz configuration](https://github.com/google/oss-fuzz/tree/master/projects/exiv2) for build instructions) as security issues.
```

Examples of issues that are **not** security issues:

* Bugs in the applications in the `samples` sub-directory. Those are demo applications that are not intended for production use.
* A bug that can only be triggered with a custom-written `main()` function.

> * A bug that can only be triggered by an unrealistic command line, such as `exiv2 <weird command line arguments generated by a python script>`.

Author: ```suggestion
* A bug that can only be triggered by an unrealistic command line, such as `exiv2 <weird command line arguments generated by a Python script>`.
```

* Performance issues, such as: "`exiv2 <very large input file>` runs slowly."

> * Sub-optimal error messages. For example, exiv2 exits with the error message: "uncaught exception".

Author: ```suggestion
* Sub-optimal error messages. For example, `exiv2` exits with the error message: "uncaught exception".
```
